doc/developers-guide/Testing-MongooseIM.md

```diff
@@ -4,55 +4,57 @@ In shell #1:
 
 ```sh
 $ cd $MONGOOSEIM
+$ ./rebar3 compile
 $ make devrel
 ```
 
 In shell #2:
 
 ```sh
-$ cd $MONGOOSEIM/dev/mongooseim_node1
+$ cd $MONGOOSEIM/_build/mim1/rel
 $ ./bin/mongooseimctl live
 ```
 
 In shell #3:
 
 ```sh
-$ cd $MONGOOSEIM/dev/mongooseim_node2
+$ cd $MONGOOSEIM/_build/mim2/rel
 $ ./bin/mongooseimctl live
 ```
 
 In shell #4:
 
 ```sh
-$ cd $MONGOOSEIM/dev/mongooseim_node3
+$ cd $MONGOOSEIM/_build/mim3/rel
 $ ./bin/mongooseimctl live
 ```
 
 In shell #5:
 
 ```sh
-$ cd $MONGOOSEIM/dev/mongooseim_fed1
+$ cd $MONGOOSEIM/_build/fed1/rel
 $ ./bin/mongooseimctl live
 ```
 
 Back to shell #1:
 
 ```sh
+$ cd test.disabled/ejabberd_tests
```

Comment:
Do you mean just `test` dir in project's root? This will confuse `rebar3`.
Ideally would be to do it as you suggest, that's probably out of the scope of this already not so small PR.